src/domain/proposal.ts

```diff
@@ -70,6 +70,9 @@ export function getProposalIPFSData(proposal: Proposal): Proposal {
         if (tagsData != null && tagsData.kind === JSONValueKind.ARRAY) {
           let tagsObjects = tagsData.toArray();
           for (let i = 0; i < tagsObjects.length; i++) {
+            if (i === 100) {
+              break;
+            }
```

Comment:
`let tagsObjects = tagsData.toArray();
  let tagsLength = tagsObjects.length < 100 ?  tagsObjects.length : 100; 
 for (let i = 0; i < tagsLength; i++) {
...
`

Reply:
cleaner ?